03.asynchronous/promise.js

```diff
@@ -0,0 +1,43 @@
+import {
+  openDatabaseConnection,
+  closeDatabaseConnection,
+  runDatabaseQuery,
+  getDatabaseData,
+} from "./db_utils.js";
+
+let newDb = undefined;
```

Comment:
`undefined` で初期化するのではなく、初期化するのをやめればその変数の値は自動的に `undefined` になるということです。

```suggestion
let newDb;
```

Reply:
自動的に`undefined`になるように修正しました。